test/cases/040_packages/023_wireguard/test-wireguard.yml

```diff
@@ -11,11 +11,11 @@ onboot:
     image: linuxkit/dhcpcd:f3f5413abb78fae9020e35bd4788fa93df4530b7
     command: ["/sbin/dhcpcd", "--nobackground", "-f", "/dhcpcd.conf", "-1"]
   - name: wg0
-    image: linuxkit/ip:b084fd2ada446015e00e43a441bb0ae41ff8b227
+    image: linuxkit/ip:722329e06dccb26348876a00ba5e2d247cad206b
     net: new
     binds:
       - /etc/wireguard:/etc/wireguard
-    command: ["sh", "-c", "ip link set dev wg0 up; ip address add dev wg0 192.168.2.1/24; wg setconf wg0 /etc/wireguard/wg0.conf; wg show wg0"]
+    command: ["/usr/bin/lk-wg-config", "configure", "/etc/wireguard/wg0.conf"]
     runtime:
       interfaces:
         - name: wg0
```

Comment:
Another way of going about this would to simply reimplement lk-wg-config in Go, and then add ` - config: /etc/wireguard/wg1.conf` to this line, and have Go handle it from the yaml. However, it's probably easiest to just do it this way with the script.

Reply:
cant do that easily as this code is run outside the container namespace, so config gets lost.